### PR TITLE
RedHat Pack

### PR DESCRIPTION
**Pull request by tkatzir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
XSIAM Modeling Rules for RedHat.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/RedHat/ModelingRules/RedHatEventsCollectorXDRC/RedHatEventsCollectorXDRC.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: RedHatEventsCollectorXDRC
+id: RedHatEventsCollectorXDRC
+fromversion: 3.3.0
```
Comment (gal-forer) [inline Packs/RedHat/ModelingRules/RedHatEventsCollectorXDRC/RedHatEventsCollectorXDRC.yml:3]: We used  6.8.0 in all the XSIAM integrations we did

File: Packs/RedHat/ModelingRules/RedHatEventsCollectorXDRC/RedHatEventsCollectorXDRC.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+rules: ''
```
Comment (gal-forer) [inline Packs/RedHat/ModelingRules/RedHatEventsCollectorXDRC/RedHatEventsCollectorXDRC.yml:4]: You do not have a schema, do you need it ?

File: Packs/RedHat/ModelingRules/RedHatEventsCollectorBrokerVM/RedHatEventsCollectorBrokerVM.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+name: RedHatEventsCollectorBrokerVM
+id: RedHatEventsCollectorBrokerVM
+fromversion: 3.3.0
```
Comment (gal-forer) [inline Packs/RedHat/ModelingRules/RedHatEventsCollectorBrokerVM/RedHatEventsCollectorBrokerVM.yml:3]: We used  6.8.0 in all the XSIAM integrations we did

File: Packs/RedHat/ModelingRules/RedHatEventsCollectorBrokerVM/RedHatEventsCollectorBrokerVM.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+rules: ''
```
Comment (gal-forer) [inline Packs/RedHat/ModelingRules/RedHatEventsCollectorBrokerVM/RedHatEventsCollectorBrokerVM.yml:4]: You do not have a schema, do you need it ?

**Review by gal-forer — CHANGES_REQUESTED** [review]

File: Packs/RedHat/ModelingRules/RedHatEventsCollectorBrokerVM/RedHatEventsCollectorBrokerVM.yml (hunk 3 above)
Comment (tkatzir) [reply Packs/RedHat/ModelingRules/RedHatEventsCollectorBrokerVM/RedHatEventsCollectorBrokerVM.yml]: https://github.com/demisto/content/pull/19252/commits/e0b068ed0f1e7c2b3cea5e2ce94c1487713c1ad1

File: Packs/RedHat/ModelingRules/RedHatEventsCollectorXDRC/RedHatEventsCollectorXDRC.yml (hunk 1 above)
Comment (tkatzir) [reply Packs/RedHat/ModelingRules/RedHatEventsCollectorXDRC/RedHatEventsCollectorXDRC.yml]: https://github.com/demisto/content/pull/19252/commits/e0b068ed0f1e7c2b3cea5e2ce94c1487713c1ad1

File: Packs/RedHat/ModelingRules/RedHatEventsCollectorBrokerVM/RedHatEventsCollectorBrokerVM.yml (hunk 4 above)
Comment (tkatzir) [reply Packs/RedHat/ModelingRules/RedHatEventsCollectorBrokerVM/RedHatEventsCollectorBrokerVM.yml:4]: https://github.com/demisto/content/pull/19252/commits/71739f96eab4b8e30e32b39f1be422038d76a882

File: Packs/RedHat/ModelingRules/RedHatEventsCollectorXDRC/RedHatEventsCollectorXDRC.yml (hunk 2 above)
Comment (tkatzir) [reply Packs/RedHat/ModelingRules/RedHatEventsCollectorXDRC/RedHatEventsCollectorXDRC.yml:4]: https://github.com/demisto/content/pull/19252/commits/71739f96eab4b8e30e32b39f1be422038d76a882

**Review by avidan-H — APPROVED** [review]

**Review by gal-forer — APPROVED** [review]

**Pull request merged**